sound/soc/sof/intel/hda-loader.c

```diff
@@ -507,3 +549,12 @@ int hda_dsp_ext_man_get_cavs_config_data(struct snd_sof_dev *sdev,
 
 	return 0;
 }
+
+int hda_dsp_core_stall_icl(struct snd_sof_dev *sdev, unsigned int core_mask)
+{
+	/* stall core */
```

Comment:
dont you need to check for core_mask and make sure they are in the host_managed_cores?

Reply:
ACK. Good point. Will include this.